tests/interactive/test_app.py

```diff
@@ -27,6 +27,15 @@
 from xdsl.utils.exceptions import ParseError
 from xdsl.utils.parse_pipeline import PipelinePassSpec, parse_pipeline
 
+# Mock pyclip_copy for testing
+_copied_text: str | None = None
+
+
+def pyclip_copy(text: str) -> None:
+    """Mock version of pyclip_copy that stores copied text for inspection during tests"""
+    global _copied_text
+    _copied_text = text
```

Comment:
Why was that necessary?

Reply:
Because pyclip is also an optional dependency, and it would be nice to actually test what the button copies, even if we don't right now. I could add this quickly

Reply:
I actually couldn't get this to work, and added a different API instead.